packages/server/src/api/controllers/application.js

```diff
@@ -63,6 +65,28 @@ function getUserRoleId(ctx) {
     : ctx.user.role._id
 }
 
+async function getAppUrlIfNotInUse(ctx) {
+  let url
+  if (ctx.request.body.url) {
+    url = encodeURI(ctx.request.body.url)
+  } else {
+    url = encodeURI(`${ctx.request.body.name}`)
+  }
+  url = `/${url.replace(/\/|\\/g, "")}`.toLowerCase()
```

Comment:
NAB: It's nice to pull the regex out into a variable so you can see what it's doing at a glance